### ISSUE TITLE
Documentation currently lacks a simple example.

### ISSUE THREAD

**Issue by chadagreene** [JuliaCI/PkgTemplates.jl#381]
As a Julia newbie who's trying to learn how to create packages the right way, I find the current documentation for PkgTemplates doesn't offer much of an on-ramp for beginners. I cannot find any sort of "Follow these steps" for a simplest-case scenario of creating a package and linking it to GitHub. Instead, the docs skip straight to "A More Complicated Example", which is presented without context or explanation. 

After tinkering a bit, I _think_ I've found a process that works, which I've done my best to document [here](https://discourse.julialang.org/t/trouble-publishing-my-first-package-to-github/93293/5?u=chadagreene). But ideally, I'm hoping that someone who knows what they're doing can add a simple Minimal Working Example to the docs, explaining each step of how to create a package the proper way.

**Comment by nsajko** [comment]
TLDR: run `generate()` and the process is foolproof and interactive

**Comment by oxinabox** [comment]
There definately used to be an example template in the docs that had some common options you might like to play with.
We should bring that example back.
It is nice to have a basic `Template` to tweak
